backend/store/issue.go

```diff
@@ -8,13 +8,28 @@ import (
 	"strings"
 	"time"
 
-	"github.com/lib/pq"
+	"github.com/go-ego/gse"
+	"github.com/jackc/pgtype"
 	"github.com/pkg/errors"
 
 	"github.com/bytebase/bytebase/backend/common"
 	api "github.com/bytebase/bytebase/backend/legacyapi"
 )
 
+var getSegmenter func() *gse.Segmenter
+
+func init() {
+	var segmenterDic gse.Segmenter
+	if err := segmenterDic.LoadDict(); err != nil {
+		panic(errors.Wrapf(err, "failed to load segmenter dictionary"))
+	}
+	getSegmenter = func() *gse.Segmenter {
+		var segmenter gse.Segmenter
+		segmenter.Dict = segmenterDic.Dict
+		return &segmenter
+	}
+}
+
```

Comment:
`LoadDict()` is time-consuming, it takes about several seconds on my computer. Fortunately, `LoadDict()` builds `segmenter.Dict` which is only read afterward, so we can build it once and pass it around by reference.